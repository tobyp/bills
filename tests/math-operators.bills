#!./bills
A	B.	0.5*2.00
C	D.	2+0.5
E	F.	5.5-9
G	H.	2/3
