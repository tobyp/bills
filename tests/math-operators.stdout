todo: B pays 1.0 to A
todo: D pays 2.5 to C
todo: E pays 3.5 to F
todo: H pays 0.6666667 to G
